pgjdbc/src/main/java/org/postgresql/Driver.java

```diff
@@ -477,8 +482,11 @@ public int getMajorVersion()
         return MAJORVERSION;
     }
 
-
-    public static final int MINORVERSION = @MINORVERSION@;
+    public static final int MINORVERSION =
```

Comment:
also keep in mind that it is much easy to exclude text blocks from result with 
```
//#-
...
//#+
```
it will be working in the same way as
```
//#if false
...
//#endif
```